lib/forms/index.js

```diff
@@ -453,19 +450,17 @@ function Forms() {
     return cb();
   };
 
-  this.emit = function (event, eventParams) {
+  this.emit = function(event, eventParams) {
     events.EventEmitter.prototype.emit.call(this, arguments);
-    _.each(this.formEventListeners, function (formEventListener) {
+    _.each(this.formEventListeners, function(formEventListener) {
       formEventListener.emit(event, eventParams);
     });
   };
 }
 
-module.exports = function (cfg) {
+module.exports = function(cfg) {
   assert.ok(cfg, 'cfg is undefined');
   config = cfg;
-  appname = cfg.fhapi.appname;
-  fhutils = new futils(config);
```

Comment:
Can you confirm that there are no usages of these removed exports? Is it possible that a user could be depending on them (I haven't checked if they're exposed beyond this module)?

Reply:
If you think that it has some reason then is better we keep it anyway.